docs/sources/tempo/setup/operator/object-storage.md

```diff
@@ -43,6 +41,72 @@ Tempo Operator supports [AWS S3](https://aws.amazon.com/), [Azure](https://azure
         type: s3
   ```
 
+### Secure Token Service (STS) Installation
+
+* Create an Object Storage secret with keys as follows:
```

Comment:
Same suggestion to number steps in procedures instead of using bullets: 

```suggestion
1 Create an Object Storage secret with keys as follows:
```